scripts/warn-outside-container

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eu
+
+target="${1:-}"
+
+if [[ -z "${DISABLE_WARN_OUTSIDE_CONTAINER:-}" ]]; then
+    (
+        echo
+        echo
+        echo "WARNING: you are not in a container."
+        echo "Use \"make -f docker.Makefile $target\" or set"
+        echo "DISABLE_WARN_OUTSIDE_CONTAINER=1 to disable this warning."
+        echo
+        echo
+    ) >&2
+    sleep 5
```

Comment:
> Maybe there's some way to have to exit, but I couldn't find a way to do it with make.

Perhaps just print `Press Ctrl+C now to abort this script.`

> It could work without parallel, but I don't see why we need to support running it without. We have a container that provides parallel, and we shouldn't support running it outside of a container.

Yes, you're right; should not be needed